core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/FixedRateSamplingTelemetryProcessor.java

```diff
@@ -157,16 +157,21 @@ public boolean process(Telemetry telemetry) {
                 if (samplingSupportingTelemetry.getSamplingPercentage() == null) {
 
                     samplingSupportingTelemetry.setSamplingPercentage(samplingPercentage);
-
-                    if (SamplingScoreGeneratorV2.getSamplingScore(telemetry) >= samplingPercentage) {
-
-                        InternalLogger.INSTANCE.info("Item %s sampled out", telemetry.getClass());
-                        return false;
-                    }
+                                        
+                    
                 } else {
                     InternalLogger.INSTANCE.info("Item has sampling percentage already set to :"
                             + samplingSupportingTelemetry.getSamplingPercentage());
+                    
+                    samplingPercentage = samplingSupportingTelemetry.getSamplingPercentage();
```

Comment:
Override the local sampling percentage with the user set sampling percentage. Else global sampling percentage will be applied even if the user has set the sampling percentage.